WISE-Community/WISE
# Added side-by-side layout to homepage library; Updated to angular 6

Also updated some UI elements (blurb, call to action) and some globaly and homepage styles/layout.

Note: This pull request includes updates to angular 6 and angular-material 6.

To test:

1. Load /site and make sure everything looks okay.
2. Try out different browser sizes to make sure the homepage and library are responsive.
3. Make sure library filter controls are hidden on smaller screens and that you can toggle them to be visible or not using the filter button next to the search input.
4. Filter toggle button should show badge when there are any active filters.
5. Make sure library search and filters work.
